README-group.md

```diff
@@ -148,6 +148,7 @@ Variable | Description | Required
 `service` | List of service name strings assigned to this group. Only usable with IPA versions 4.7 and up. | no
 `membermanager_user` | List of member manager users assigned to this group. Only usable with IPA versions 4.8.4 and up. | no
 `membermanager_group` | List of member manager groups assigned to this group. Only usable with IPA versions 4.8.4 and up. | no
+`external_member` | List of embers of a trusted domain in DOM\\name or name@domain form. | no
```

Comment:
Think there's a typo in that the 3rd word should be 'members' not 'embers'

Reply:
fixed.